src/main/kotlin/widgets/svgButton/view/SvgButton.kt

```diff
@@ -0,0 +1,82 @@
+package widgets.svgButton.view
+
+import afester.javafx.svg.SvgLoader
+import tornadofx.*
+import javafx.scene.Group
+import javafx.scene.layout.StackPane
+import javafx.scene.paint.Paint
+
+
+//Class to create circle button with SVG graphic inside
+//Use "::functionName" to pass function as parameter
+//Example how to use: SvgButton(64.0, c("#CC4141"), "forwardArrowIcon", operation = ::abc)
+//For suggestions the best values to put in the parameter for certain buttons, scroll to bottom of this file
+class SvgButton(var buttonSize: Double= 64.0,
+                var myVariable: Paint = c("#ffffff"),
+                var svgFileName: String,
+                var svgScaleXCompareToButton: Double = 0.50484,
+                var svgScaleYCompareToButton: Double = 0.50484,
```

Comment:
size, color, scaling, should be done through styling not through constructor arguments

Reply:
If the constructor doesn't have styling arguments, the SvgButton can't be reused as different buttons